mobile/src/modules/auth/components/details/TransactionListItem.tsx

```diff
@@ -58,6 +60,9 @@ const styles: any = {
   outcomingAmount: {
     marginLeft: Sizes.MEDIUM,
     flexDirection: 'row'
+  },
+  account: {
+    flexWrap: 'wrap',
   }
```

Comment:
Fixes an issue where some transaction details would be hidden if a user has a custom font size or font style set (reported by paulpoco).

Reply:
![image](https://user-images.githubusercontent.com/42594751/105609074-99e10480-5d5b-11eb-8ef0-f9a31323b35a.png)